roles/solr_collection/README.md

```diff
@@ -1,38 +1,20 @@
-Role Name
-=========
+solr_collection
+===============
 
-A brief description of the role goes here.
-
-Requirements
-------------
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+Solr collection creation and management for Solr Cloud with Zookeeper.
 
 Role Variables
 --------------
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+* `solr_server`: server where configset directory will be created, and where zookooper will be run
+* `num_shards`: number of shards; default  3
```

Comment:
You might not need more than 1. We moved even the catalog to 1 shard to make indexing faster.